### DISCUSSION TITLE
如何设置使空闲时mpv启动大小固定为特定大小，播放时启动大小为片源大小？

### DISCUSSION THREAD

**Discussion by shanyan-wcx** [hooke007/mpv_PlayKit#344] [mpv 通用]
如题。

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/discussions/69

**Reply by shanyan-wcx** [reply]
没看懂要怎么搞，能仔细讲讲吗

**Reply by hooke007** [reply]
> 空闲时mpv启动大小固定为特定大小

`--geometry=1280x720`

**Reply by shanyan-wcx** [reply]
加上geometry=1280x720后打开视频也是这个大小了

**Reply by hooke007** [reply]
是，文档就是这么记录的。两者做不到同时满足
有这个需求的话建议向mpv提fr
